newdle/client/src/components/answer/AnswerPage.js

```diff
@@ -109,78 +110,96 @@ export default function AnswerPage() {
     return null;
   }
 
+  const {final_dt: finalDate} = newdle;
+
   return (
     <div>
       <Grid container>
-        {error && (
+        {finalDate ? (
```

Comment:
What about moving this outside this `return`? ie `if (finalDate) { return ...; }`?

that avoids this massive indentation change and i think it's also clearer because it's completely separate from the rest here